posts/templates/posts/post_confirm_delete.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+{% load static %}
+{% block styles %}
+    <link rel="stylesheet" type="text/css" href="{% static 'posts/forms.css' %}">
+{% endblock styles %}
+{% block content %}
+    <div class="content-section">
+        <form method="POST" enctype="multipart/form-data">
+            {% csrf_token %}
+            <fieldset class="form-group">
+                <legend class="border-bottom text-center mx-auto">Delete Post</legend>
+                <h2>Are you sure you want to delete the post?</h2>
+            </fieldset>
+            <div class="form-group">
+                <button class="btn btn-outline-danger" type="submit">Yes, Delete</button>
```

Comment:
I think it should be something more like 

```suggestion
                  <button class="btn btn-outline-danger" type="submit">Confirm</button>
```

Reply:
I think that "Yes, Delete" is a more suitable answer to the question: "Are you sure you want to delete this post?" than "Confirm".